addons/mail/wizard/mail_resend_message.py

```diff
@@ -12,13 +12,18 @@ class MailResendMessage(models.TransientModel):
     mail_message_id = fields.Many2one('mail.message', 'Message', readonly=True)
     partner_ids = fields.One2many('mail.resend.partner', 'resend_wizard_id', string='Recipients')
     notification_ids = fields.Many2many('mail.notification', string='Notifications', readonly=True)
+    can_resend = fields.Boolean(compute='_compute_can_resend')
```

Comment:
Maybe we could rename has_cancel to can_cancel then, in order to keep a coherent naming ? Would imply a migration script ^^ .